internal/deploy/docker_client.go

```diff
@@ -311,6 +311,10 @@ func InspectContainer(service ServiceRequest) (*types.ContainerJSON, error) {
 		}).Fatal("Cannot list containers")
 	}
 
+	if len(containers) == 0 {
```

Comment:
We have a lot of usages for the `.Inspect` method where we simply discard the raised error. We need to think about changing that pattern